aptos-move/move-examples/large_packages/sources/large_packages.move

```diff
@@ -4,11 +4,14 @@
 /// Currently this module does not support modules that are larger than 63KB as that is the maximum
 /// that can fit within a transaction and this framework does not split up individual modules.
```

Comment:
ah now I understand why I was confused today about the max size...you were referring to it supporting packages of any size, but the module size is still limited. although even that is only bc this doesn't handle splitting them up for you, right?

Reply:
well it was... but I didn't update the comments...

I generalized the metadata chunking logic and then applied it to modules, so now it supports any size.

Reply:
ahh right, you'd just have to call it multiple times